.babelrc

```diff
@@ -1,3 +1,27 @@
 {
-  "presets": ["stage-1", "es2015", "react"]
+  "presets": [
+    [
+      "env",
+      {
+        "modules": false,
+        "loose": true
+      }
+    ],
+    "stage-1",
```

Comment:
Personally I prefer `stage-3` or `stage-4` as they are more stable, but It depends